### PR TITLE
Desvelted

### PR DESCRIPTION
**Pull request by 4194304**
functions pretty much the exact same, code needs to be cleaned up though

### CODE REVIEW

**Comment by SolDev69** [comment]
Will not merge.

**Comment by SolDev69** [comment]
This the right pr? I'll review this tomorrow

**Comment by 4194304** [comment]
everything is set up so no problems on that part

**Comment by 4194304** [comment]
Repository is up to date now

File: css/stylesheet0.css
Comment (SolDev69) [inline css/stylesheet0.css:1]: half of these are ripped from svelte

File: css/stylesheet8.css
Comment (SolDev69) [inline css/stylesheet8.css:1]: Ripped from svelte

File: css/stylesheet9.css
Comment (SolDev69) [inline css/stylesheet9.css:1]: ripped from svelte

**Review by SolDev69 — CHANGES_REQUESTED** [review]
You just ripped a bunch of stuff from inspect element 🤦‍♂️

File: css/stylesheet0.css
Comment (4194304) [reply css/stylesheet0.css:1]: I know that

File: css/stylesheet9.css
Comment (4194304) [reply css/stylesheet9.css:1]: I can merge the stylesheets

**Comment by 4194304** [comment]
Cleaned up code

**Pull request closed without merging**